### ISSUE TITLE
TypeError: Only messages, members or roles may be passed

### ISSUE THREAD

**Issue by tekulvw** [Cog-Creators/Red-DiscordBot#1179]
https://sentry.io/will-tekulve/bot-development/issues/419244737/

```
TypeError: Only messages, members or roles may be passed
  File "redbot/core/utils/mod.py", line 132, in is_mod_or_superior
    raise TypeError('Only messages, members or roles may be passed')

Exception in on_message
```

**Comment by palmtree5** [comment]
This should be a fairly easy fix

**Comment by Tobotimus** [comment]
Fixed in #1185
